lib/watchOrServe.js

```diff
@@ -6,16 +6,27 @@ module.exports = function ( gobblefile, getTask ) {
 		resuming;
 
 	resume();
+logger.info({message: 'gobblefile is ' + gobblefile});
+	function watch () {
+		pathwatcher.watch( gobblefile, type => {
 
-	pathwatcher.watch( gobblefile, type => {
-		if ( type === 'change' ) restart();
-	});
+			if ( type === 'change' ) {
+				logger.info({ code: 'GOBBLEFILE_CHANGED' });
+				return restart();
+			}
+
+			if ( type === 'delete' ) {
+				logger.info({ code: 'GOBBLEFILE_DELETED' });
+				return setTimeout( watch, 2000 );
```

Comment:
I don't have a strong opinion about this particular bit. Waiting 2 seconds works reliably for my use case, but I can see the appeal of checking for the gobblefile in a loop.